packages/taquito-utils/src/taquito-utils.ts

```diff
@@ -289,7 +289,7 @@ export const mic2arr = function me2(s: any): any {
  *
  * @param buffer Buffer to convert
  */
-export const buf2hex = (buffer: Buffer): string => {
+export const buf2hex = (buffer: Uint8Array): string => {
```

Comment:
buf2hex is a public export function the param type change looks like a breaking change, might wan to reassess impact

Reply:
From my understanding, we should be using `Uint8Array`: https://stackoverflow.com/a/54814368/742775